content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -99,5 +99,29 @@ where payments will be made (the URLs of the servers hosting your Odoo instances
    :align: center
    :alt: Allows payments originated from a specific domain.
 
+Capture the payment after the delivery
+--------------------------------------
+
+If you wish to manually capture an amount instead of having an immediate capture, Adyen allows it.
+
+You have to enable it both on Odoo, as explained :ref:`here <payment_acquirers/capture_amount>`,
+and on Adyen.
```

Comment:
@LoredanaLrpz 's suggestion is good but I would also request to have the link on the relevant text/title, rather than on "here". it helps the reader to know directly what the link is about, and it is great for SEO improvement.